.env.example.prod

```diff
@@ -0,0 +1,31 @@
+USE_DOCKER=yes
+IPYTHONDIR=/app/.ipython
+
+DJANGO_SECRET_KEY=terriblekey
+DJANGO_ALLOWED_HOSTS=pra-tracker.tech-bloc-sea.dev
+
+
+POSTGRES_HOST=postgres
+POSTGRES_PORT=5432
+POSTGRES_DB=pra_request_tracker
+POSTGRES_USER=sjxHBcWufSbTJRVSCynqDavkxHefYxJP
+POSTGRES_PASSWORD=wgnuhcBthzKLm6e9NCUgMZXVlwjvEWNAzTqqA3LAmN7LcXzWmOWxjEgogCQghVMo
+
+AWS_STORAGE_BUCKET_NAME=prr-tracker
+AWS_ACCESS_KEY_ID=minio
+AWS_SECRET_ACCESS_KEY=minio123
+AWS_S3_REGION_NAME=us-west-2
+AWS_S3_ENDPOINT_URL=http://minio:9000
+MINIO_ROOT_PASSWORD=minio123
```

Comment:
Should we maybe change these to AWS/DO Space values? And do we need the `MINIO_ROOT_PASSWORD` variable for prod?

Reply:
Oh you're right, I'll change that